it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -56,94 +56,94 @@
 12505=Caricamento allegato in corso…
 
 ## @name NS_MSG_COULDNT_OPEN_FCC_FOLDER
-12506=Impossibile aprire la cartella Posta inviata. Verificare che le impostazioni di posta siano corrette.
+12506=Impossibile aprire la cartella Posta inviata. Verifica che le tue impostazioni di posta siano corrette.
 
 ## @name NS_MSG_CANT_POST_TO_MULTIPLE_NEWS_HOSTS
-12507=Non è possibile inviare verso più server di gruppi di discussione. Riprovare.
+12507=Non è possibile pubblicare su host di notizie multipli. Riprova.
 
 ## @name NS_MSG_ASSEMB_DONE_MSG
-12508=Ricomposizione messaggio…completato.
+12508=Unione messaggio in corso… Completata.
 
 ## @name NS_MSG_ASSEMBLING_MSG
-12509=Ricomposizione messaggio…
+12509=Unione messaggio in corso…
 
 ## @name NS_MSG_NO_SENDER
-12510=Non è stato specificato il mittente. Inserire il proprio indirizzo e-mail nelle impostazioni di Posta e gruppi di discussione.
+12510=Non è stato specificato il mittente. Inserisci il tuo indirizzo e-mail nelle impostazioni account E-mail e Newsgroup.
 
 ## @name NS_MSG_NO_RECIPIENTS
-12511=Non sono stati specificati destinatari. Inserire un destinatario o un gruppo di discussione nella sezione degli indirizzi.
+12511=Non sono stati specificati destinatari. Inserisci un destinatario o un newsgroup nell'area indirizzo.
 
 ## @name NS_MSG_ERROR_WRITING_FILE
 12512=Errore di scrittura sul file temporaneo.
 
 ## @name NS_ERROR_SENDING_FROM_COMMAND
-12514=Si è verificato un errore durante l'invio della posta. Il server di posta ha risposto: %s. Verificare che il proprio indirizzo e-mail nelle impostazioni di posta sia corretto e riprovare.
+12514=Si è verificato un errore durante l'invio della e-mail. Il server di posta ha risposto: %s. Verifica nelle impostazioni e-mail che il tuo indirizzo e-mail sia corretto e riprova.
 
 ## @name NS_ERROR_SENDING_DATA_COMMAND
-12516=Si è verificato un errore (SMTP) durante l'invio della posta. Il server ha risposto: %s.
+12516=Si è verificato un errore (SMTP) durante l'invio della e-mail. Il server ha risposto: %s.
 
 ## @name NS_ERROR_SENDING_MESSAGE
-12517=Si è verificato un errore durante l'invio della posta. Il server di posta ha risposto: %s. Controllare il messaggio e riprovare.
+12517=Si è verificato un errore durante l'invio della e-mail. Il server di posta ha risposto: %s. Controlla il messaggio e riprova.
 
 ## @name NS_ERROR_POST_FAILED
-12518=Il messaggio non può essere pubblicato a causa della mancata connessione al server delle news. Il server potrebbe non essere raggiungibile o rifiutare la connessione. Ricontrollare le impostazioni del server delle news o contattare il proprio amministratore di sistema.
+12518=Il messaggio non può essere pubblicato poiché la connessione al server delle news non è riuscita. Il server potrebbe non essere disponibile o rifiutare la connessione. Verifica che le tue impostazioni del server news siano corrette oppure contatta il tuo Amministratore di sistema.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12518=Il messaggio non può essere pubblicato poiché la connessione al server delle news non è riuscita. Il server potrebbe non essere disponibile o rifiutare la connessione. Verifica che le tue impostazioni del server news siano corrette oppure contatta l'Amministratore del sistema.